src/soilbiogeochem/SoilBiogeochemVerticalProfileMod.F90

```diff
@@ -62,7 +62,7 @@ subroutine SoilBiogeochemVerticalProfile(bounds, num_soilc,filter_soilc,num_soil
     integer                         , intent(in)    :: num_soilp                               ! number of soil patches in filter
```

Comment:
Looks like these changes could be dropped

Reply:
Two empty comment lines were removed. Do we need keep it?

Reply:
done